generators/src/main/resources/typescript/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@pdc/sdk",
+  "version": "0.0.0",
+  "description": "A swagger client for the Philanthropy Data Commons (PDC)",
+  "author": "Open Tech Strategies",
+  "keywords": [
+    "typescript",
+    "Philanthropy Data Commons"
+  ],
+  "license": "AGPL-3.0",
+  "main": "./dist/index.js",
+  "scripts" : {
+		"build": "npm run cleanup && npm run build:node",
+		"build:node": "tsc -p tsconfig.json",
+		"cleanup": "rimraf dist"
+  },
+  "devDependencies": {
+		"@tsconfig/node18": "^18.2.2",
+		"rimraf": "^5.0.5",
```

Comment:
It might be overkill to use rimraf here, but I do like the idea of ensuring build steps are ALWAYS coming from a clean slate (the overkill is that since this is a generator, `build` is probably only going to be called once per generation, and the act of generating itself already cleans up all generated code).

That said, it's possible someone might set up a flow of:

1. Generate the code
2. Move the code
3. Invoke the build

in which case it's possible rimraf would serve a purpose (since the code is MOVED to a consistent location before the build)

Just wanted to point this out.